packages/eslint-config-react-app/index.js

```diff
@@ -23,37 +23,18 @@
 // To use them, explicitly reference them, e.g. `window.name` or `window.status`.
 const restrictedGlobals = require('confusing-browser-globals');
 
-module.exports = {
-  root: true,
-
-  parser: 'babel-eslint',
-
-  plugins: ['import', 'flowtype', 'jsx-a11y', 'react', 'react-hooks'],
-
-  env: {
-    browser: true,
-    commonjs: true,
-    es6: true,
-    jest: true,
-    node: true,
-  },
+const resolve = require('resolve');
+const path = require('path');
+const fs = require('fs');
 
-  parserOptions: {
-    ecmaVersion: 2018,
-    sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true,
-    },
-  },
+const overrides = [];
 
-  settings: {
-    react: {
-      version: 'detect',
-    },
-  },
-
-  overrides: [
-    {
+// Lint tsx only if typescript is installed.
```

Comment:
Probably it's not worth it since if you have `ts` files, you already have to install typescript:

https://stackoverflow.com/questions/60134596/create-react-app-without-typescript-got-error-failed-to-load-parser-types